core/src/main/java/org/mskcc/cbio/portal/scripts/ImportTabDelimData.java

```diff
@@ -123,8 +124,8 @@ public void importData(int numLines) throws IOException, DaoException {
                                 && geneticProfile.getGeneticAlterationType() == GeneticAlterationType.GENESET_SCORE
                                 && parts[0].equalsIgnoreCase("geneset_id");
         boolean treatmentProfile = geneticProfile!=null
```

Comment:
should we also rename `treatmentProfile` to `genericAssayProfile`?

Reply:
We agreed not to start renaming all the references to treatments in the current implementation. I have updated the reference to `treatmentProfile` in this file.